charts/sombra/CHANGELOG.md

```diff
@@ -53,3 +53,9 @@
 * Created serviceaccount.yaml template for conditional service account creation
 * Updated deployment to conditionally use the specified service account
 * **Non-breaking change**: Service account functionality is completely opt-in and disabled by default
+
+## 0.6.5
+
+* Added support for client-managed secrets via `envFrom` configuration
+* Allow loading environment variables from ConfigMaps and Secrets using the `envFrom` field
+* **Non-breaking change**: `envFrom` functionality is completely opt-in and disabled by default
```

Comment:
Super nitpicky

```suggestion
* **Non-breaking change**: `envFrom` functionality is completely opt-in and is an empty list by default
```

Reply:
Thanks! I actually think this is most correct because in Helm, empty collections [resolve to false](https://helm.sh/docs/chart_template_guide/control_structures/#ifelse), so this

```yaml
          {{- if .Values.envFrom }}
          envFrom:
            {{- toYaml .Values.envFrom | nindent 12 }}
          {{- end }}
```

will resolve to empty string when `Values.envFrom` is `[]`